test/UnitTest/NGSI-LD/test_casesNGSI-LD.py

```diff
@@ -16,899 +16,1091 @@
 
 #testCase 8
 '''
-  Test without passing Link Header 
+  To test create entity without passing Link Header 
 '''
 def test_case8():
         url=brokerIp+"/ngsi-ld/v1/entities/"
         headers={'Content-Type' : 'application/json','Accept':'application/ld+json'}
 	r=requests.post(url,data=json.dumps(ld_data.subdata),headers=headers)
-        print(r.content)
-        print(r.status_code)
+        #print(r.content)
+        #print(r.status_code)
         assert r.status_code == 201
 
 #testCase 9
 '''
-  Testing Update entity with different header and posting duplicate attribute
+  To test Update entity with two  header namely Content Type and Accept header  and posting duplicate attribute
 '''
 def test_case9():
 	url=brokerIp+"/ngsi-ld/v1/entities/urn:ngsi-ld:Vehicle:A100/attrs"
         headers={'Content-Type' : 'application/json','Accept':'application/ld+json'}
         r=requests.post(url,data=json.dumps(ld_data.subdata14),headers=headers)
-        print(r.content)
-        print(r.status_code)
+        #print(r.content)
+        #print(r.status_code)
         assert r.status_code == 207
 
 #testCase 10
 '''
-  Testing Update entity with wrong id
+  To test Update entity with wrong id
 '''
 def test_case10():
         url=brokerIp+"/ngsi-ld/v1/entities/urn:ngsi-ld:Vehicle:A800/attrs"
         headers={'Content-Type' : 'application/json','Accept':'application/ld+json'}
         r=requests.post(url,data=json.dumps(ld_data.subdata14),headers=headers)
-        print(r.content)
-        print(r.status_code)
+        #print(r.content)
+        #print(r.status_code)
         assert r.status_code == 404
 
 #testCase 11
 '''
-  Testing Update entity with wrong id and not passing Accept Header
+  To test Update entity with wrong id and not passing Accept Header
 '''
 def test_case11():
         url=brokerIp+"/ngsi-ld/v1/entities/urn:ngsi-ld:Vehicle:A800/attrs"
         headers={'Content-Type' : 'application/json'}
         r=requests.post(url,data=json.dumps(ld_data.subdata14),headers=headers)
-        print(r.content)
-        print(r.status_code)
+        #print(r.content)
+        #print(r.status_code)
         assert r.status_code == 404
 
 #testCase 12
 '''
-  Testing Update entity with different header
+  To test Update entity with three  header namely Content-Type, Accept and context Link 
 '''
 def test_case12():
         url=brokerIp+"/ngsi-ld/v1/entities/urn:ngsi-ld:Vehicle:A4580/attrs"
         headers={'Content-Type' : 'application/json','Accept':'application/ld+json','Link':'<{{link}}>; rel="https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld"; type="application/ld+json"'}
         r=requests.post(url,data=json.dumps(ld_data.subdata14),headers=headers)
-        print(r.content)
-        print(r.status_code)
+        #print(r.content)
+        #print(r.status_code)
         assert r.status_code == 204
 
 #testCase 13
 '''
-  Testing Update entity with different headers and passing inappropriate payload
```

Comment:
What is the meaning of different header values in this test case and other similar test case titles?